pyaedt/edb_core/siwave.py

```diff
@@ -405,9 +405,281 @@ def _db(self):
         return self.parent.db
 
     @aedt_exception_handler
-    def create_circuit_port(self, positive_component_name, positive_net_name, negative_component_name=None,
-                              negative_net_name="GND", impedance_value=50, port_name=""):
-        """Create a circuit port.
+    def _create_terminal_on_pins(self, source):
+        """Create a terminal on pins.
+
+        Parameters
+        ----------
+        source : ``VoltageSource`` or ``CircuitPort`` or ``CurrentSource`` or ``ResistorSource``
```

Comment:
These don't have to be spelled out as literals here since numpydoc is expecting class names.
```suggestion
        source : VoltageSource, CircuitPort, CurrentSource or ResistorSource
```